lib/tessel/deploy.js

```diff
@@ -528,6 +604,21 @@ actions.resolveBinaryModules = function(opts) {
   });
 };
 
+actions.resolveBinaryModules.readGypFileSync = function(gypfile) {
+  var python = process.env.PYTHON || 'python';
+  var program = `import ast, json; print json.dumps(ast.literal_eval(open("${gypfile}").read()));`;
```

Comment:
So we've got Python code inside a JavaScript library to pull in a binary likely written in C++/C. Nice.